bpython/importcompletion.py

```diff
@@ -48,6 +49,8 @@
 
 # The cached list of all known modules
 modules = set()
+#List of stored paths
```

Comment:
Missing space after #

It would also be good to have some documentation here why the paths are stored. I'd also note here that this handles symlinks, but not mount points.